pkgs/applications/networking/geph5/default.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libxkbcommon,
+  openssl,
+  jemalloc,
+  sqlite,
+  vulkan-loader,
+  wayland,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "geph5";
+  version = "0.2.55";
+
+  src = fetchFromGitHub {
+    owner = "geph-official";
+    repo = "geph5";
+    rev = "geph5-client-v${version}";
```

finalAttrs is generally preferred
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
  pname = "geph5";
  version = "0.2.55";

  src = fetchFromGitHub {
    owner = "geph-official";
    repo = "geph5";
    rev = "geph5-client-v${finalAttrs.version}";
```